test/test_sparse_csr.py

```diff
@@ -1049,6 +1049,118 @@ def test_resize(self, device, dtype):
             self.assertEqual(a.shape, (new_shape[-2], new_shape[-1]))
             self.assertEqual(a._nnz(), 5)
 
+    @skipMeta
+    @dtypes(torch.float, torch.bool)
+    @all_sparse_compressed_layouts('layout_a')
+    @all_sparse_compressed_layouts('layout_b')
+    def test_resize_as_sparse_compressed(self, device, dtype, layout_a, layout_b):
+
+        def _check_resize_b_as_a(b, a, should_resize=True):
+            br = b.clone()
+            br.resize_as_sparse_(a)
+
+            # layout and shape are inherited from a
+            self.assertEqual(a.shape, br.shape)
+            self.assertEqual(a.layout, br.layout)
```

Comment:
Hmm, I missed that during my last round. But inheriting `layout` from `a` is something we really want?

Reply:
@amjames, you marked this as resolved. If this is something we still keep, let's add a comment that raises this concern?